InterneTaakAfhandeling.Web.Client/src/utils/dateUtils.ts

```diff
@@ -0,0 +1,10 @@
+export const formatDate = (dateString: string | undefined): string => {  
```

Comment:
We can make this function a bit more flexible:
```suggestion
export const formatDate = (dateString: string | number | Date | null | undefined): string => {  
```